12LessonTwelve/homework/css/style.css

```diff
@@ -0,0 +1,118 @@
+h1 {
+    padding-top: 30px;
+    font-size: 48px;
+    font-weight: 500;
+    text-align: center;
+    color: #3a4543;
+    font-family: 'Lily Script One', cursive;
+}
+section {
+    padding-top: 60px;
+}
+section div {
+    text-align: center;
+}
+img {
+    width: 300px;
+    height: 300px;
+}
+.overallbox:hover .price-box {
+    background-color: #3A4543;
+}
+.overallbox:hover .price-box-3 {
+    background-color: #3A4543;
+}
+.overallbox:hover img {
+    transform: scale(1.1);
+    transition: all 500ms ease;
+    vertical-align: middle;
+}
+.product img:hover, .onsale img:hover {
+    transform: scale(1.1);
+    transition: all 500ms ease;
+    max-width: 100%;
+    transition-property: all;
+    transition-duration: 500ms;
+    transition-timing-function: ease;
+    transition-delay: initial;
+}
+.product, .overallbox:hover {
+    overflow: hidden;
+}
+.onsale {
+    min-height: 20px;
+    min-width: 40px;
+    left: 15px;
+    padding: 4px 8px;
+    background: #F04933;
+    position: absolute;
+    color: white;
+    z-index: 100;
+}
+.price-box {
+    background-color: #66D9C1;
+    color: white;
+    border: none;
+    height: 30px;
+    width: 150px;
+    position: absolute;
+    margin: -35px 150px auto;
+    font-weight: bolder;
+    font-size: 20px;
+}
+.price-box:hover {
+    background-color: #3A4543;
+    color: white;
+    height: 30px;
+    width: 150px;
+    font-weight: bolder;
+    font-size: 20px;
+}
+.price-box-3 {
+    background-color: #66D9C1;
+    color: white;
+    border: none;
+    height: 30px;
+    width: 160px;
+    position: absolute;
+    margin: -35px 140px auto;
+    font-weight: bolder;
+    font-size: 20px;
+}
+.price-box-3:hover {
+    background-color: #3A4543;
+    color: white;
+    height: 30px;
+    width: 160px;
+    margin: -35px 140px auto;
+    font-weight: bolder;
+    font-size: 20px;
+}
```

Comment:
when you use a pseudoselector such as :hover, you don't need to copy in the styling from the original (height, width, margin, etc). You only need to specify the styling that you want to change - in this case, `background-color`.